lib/src/platform_mq.dart

```diff
@@ -85,7 +89,7 @@ class PlatformMQImpl implements PlatformMQ {
 
     Subscription? subscription = _client.subscribe(topic, qosLevel);
     if (subscription == null) {
-      _handleSubscribeFail(topic);
+      throw const PlatformUnknownException('Topic subscription failed');
```

Comment:
I spotted this :bug: while I was in here. `_handleSubscribeFail` just logs right now, so we may have been missing errors.